content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -0,0 +1,219 @@
+---
+reviewers:
+- verb
+- yujuhong
+title: Ephemeral Containers
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+This page provides an overview of ephemeral containers: a special type of container
+that runs temporarily in an existing pod to accomplish user-initiated actions such
+as troubleshooting. You use ephemeral containers to inspect services rather than
+build applications.
```

Comment:
```suggestion
to build applications.
```
(I think this will help localizers and readers for whom English is not their first language)

Reply:
SG, thanks!